sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/NetworkStateHelper.java

```diff
@@ -190,8 +191,7 @@ private boolean isAnyNetworkConnected() {
     @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
     private synchronized void onNetworkAvailable(Network network) {
```

Comment:
This synchronized can be removed.

Reply:
Shouldn't we keep it? It calls `notifyNetworkStateUpdated(true);` in the method.

Reply:
See https://github.com/Microsoft/AppCenter-SDK-Android/pull/983/files#r262664710